packages/react-hooks/src/hooks/mounted-ref.ts

```diff
@@ -3,9 +3,11 @@ import {useRef} from 'react';
 import {useIsomorphicLayoutEffect} from './isomorphic-layout-effect';
 
 export function useMountedRef() {
-  const mounted = useRef(true);
+  const mounted = useRef(false);
```

Comment:
What's the thinking behind changing this default value.

My initial thoughts would be I think we want this to retain true as its default value. This would be the value that is used between the first render when useIsomorphicLayoutEffect is called.

On a server, useIsomorphicLayoutEffect uses a plain useEffect (as useLayoutEffect errors on the  server), which runs after initial rendering so there would be a moment when this would return false, which would be different to current behaviour.